index.html

```diff
@@ -1261,7 +1259,7 @@ <h3 id="navigation-scope-security-considerations">
         </p>
         <p>
           A <a>manifest</a> is obtained and applied regardless of the
-          <a><code>media</code></a> attribute of the <a><code>link</code>
+          <a>media</a> attribute of the <a><code>link</code>
```

Comment:
Nit: seems you missed a closing `</a>`. Also, please remove the `<code>`.

Reply:
its there :-) comes a bit later